src/main/java/fopbot/KarelWorld.java

```diff
@@ -655,4 +664,23 @@ protected void loadImagesIfNotLoaded() {
         }
         imagesLoaded = true;
     }
+
+    /**
+     * Returns the {@link GuiPanel} of this world.
+     *
+     * @return the {@link GuiPanel} of this world
+     */
+    public GuiPanel getGuiPanel() {
+        return guiGp;
+    }
+
+    /**
+     * Sets the {@link GuiPanel} of this world.
+     *
+     * @param guiPanel the {@link GuiPanel} of this world
+     * @apiNote This method should not be called by students and is thought for testing purposes only.
```

Comment:
Use `@ApiStatus.Internal` annotation to trigger IDE warnings

Reply:
fixed by https://github.com/tudalgo/fopbot/pull/32/commits/3a1a07b7fd7b511297319a18be02265157a08b45